json/static-search.json

```diff
@@ -0,0 +1,7 @@
+---
+---
+
+{
+  "glossary": {{ site.data.glossary.items | jsonify }},
```

Comment:
not sure, you might need a `| safe` or something here so that html-ish entities don't get escaped. At leas that's what it's called in django.

Reply:
Cool. Added to icebox. 👍